azure_monitor/python_logger_opencensus_azure/monitoring/examples/api_4.py

```diff
@@ -0,0 +1,49 @@
+"""REST API Module using AppLogger"""
+
+import json
+import logging
+from flask import Flask, jsonify
+import sys 
+import os
+import time
+sys.path.append(os.path.join(os.getcwd(),'monitoring'))
+
+from src.logger import AppLogger
+component_name ="API_4"
```

Comment:
What is API 4?

Reply:
It doesn't appear that any special api library is being used, so this name confused me.

Reply:
This is kept to keep in sync with the other 3 examples - api_1.py , api_2.py and api_3.py.
Let me know if you would wish to have any alternative ?

The same line in other example-
component_name ="API_3"